codegen/src/generator/rest_json.rs

```diff
@@ -77,7 +77,7 @@ impl GenerateProtocol for RestJsonGenerator {
                 modify_endpoint_prefix = generate_endpoint_modification(service).unwrap_or("".to_owned()),
                 http_method = operation.http.method,
                 error_type = operation.error_type_name(),
-                status_code = operation.http.response_code.unwrap_or(200),
+                status_code = http_code_to_status_code(operation.http.response_code).unwrap_or("StatusCode::Ok".to_string()),
```

Comment:
With the implementation of `http_code_to_status_code` below, I think the only possible outcome of any status code here is "StatusCode::Ok" ?